src/lib/Microsoft.Health.Events/EventConsumers/Service/Infrastructure/EventQueueWindow.cs

```diff
@@ -0,0 +1,34 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+
+namespace Microsoft.Health.Events.EventConsumers.Service.Infrastructure
+{
+    public class EventQueueWindow
```

Comment:
Not sure of the value of this class.  Seems like this can be easily tracked on the EventQueue class.  You only ever have one instance of this per queue.